RELEASING.adoc

```diff
@@ -1,31 +1,28 @@
 = Spring Cloud GCP release process
 
-The actual release process is triggered by the release admin.
+The actual release process is triggered by the release admin and happens together with the rest of the Spring Cloud release.
 In order to simplify that process, here is a list of prerequisites to be performed before asking the release admin to run the release scripts.
 
 . Manually change the value of `TRACKING_HEADER_PROJECT_VERSION` in link:spring-cloud-gcp-core/src/main/java/org/springframework/cloud/gcp/core/UsageTrackingHeaderProvider.java[`UsageTrackingHeaderProvider`] to the version being released from the interim `BUILD-SNAPSHOT` value.
 
-. Change the value of the `<version>` elements of the `pom.xml` of all modules to the name of the version being released from the interim `BUILD-SNAPSHOT` values. This can be done using the https://www.mojohaus.org/versions-maven-plugin/set-mojo.html[mvn versions:set] command.
+. Ensure that modules that shouldn't be released have the `maven-deploy-plugin` configuration set to `<skip>true</skip>`.
```

Comment:
I believe this has to be done in the boundaries of the `central` profile as @spencergibb mention on the call.

Or am I missing anything?

Reply:
I guess that will allow us to make the code change now, instead of flipping back and forth for GA releases. I'll make the change on this PR.